jxls/src/main/java/org/jxls/builder/xls/XlsCommentAreaBuilder.java

```diff
@@ -219,11 +224,104 @@ public static void addCommandMapping(String commandName, Class<? extends Command
         return userAreas;
     }
 
+    private void extractLiterals(CellData cellData, String text, List<String> literalList) {
+		String literal = "";
+		for (int i = 0, n = text.length(); i < n; i++) {
+
+			// if the first 3 chars of literal are 'jx:'
+			if (literal.length() >= 3
```

Comment:
maybe a constant or so for the "3" ..? COMMAND_PREFIX.length or COMMAND_PREFIX_LEN
After that you can remove the comment "// if the first 3 chars of literal are 'jx:'"